internal/pkg/ssm/command.go

```diff
@@ -94,7 +94,7 @@ func RunCommand(session *session.Session, logger logr.Logger, instanceId, comman
 
 	logger.Info("Waiting for ssm command to finish")
 	var commandOut *ssm.GetCommandInvocationOutput
-	r := retrier.New(300*time.Minute, retrier.WithMaxRetries(2160, 60*time.Second))
+	r := retrier.New(310*time.Minute, retrier.WithMaxRetries(2160, 60*time.Second))
```

Comment:
I think we also need to update the `SendCommandInput`
I think we want `go test timeout` < `SSM command timeout` < `timeout in retrier that pulls the SSM command results`.
That guarantees the SSM command always gets the test results and that we always get the results of the SSM command.

Reply:
Is this what you meant? https://github.com/aws/eks-anywhere/pull/5642/files#diff-ec745d2a43e1da6e449129ed0106f64d40ac71e38690f7970f0d3d440ed44889R125-R130